tests/unit/test_class_subcmd.py

```diff
@@ -106,31 +106,32 @@ class hierarchy defined by `CLASSNAME` argument if provided.
 
   The output format is defined by the output-format general option.
 
-  The includeclassqualifiers, includeclassorigin options define optional
+  The local-only and no-qualifiers options define optional
   information to be included in the output.
 
-  The deepinheritance option defines whether the complete hiearchy is
+  The deep-inheritance option defines whether the complete class hiearchy is
```

Comment:
Show option name deep-inheritance with leading dashes.

Reply:
Fixed in _cmd_class.py